components/Pledge/Form.js

```diff
@@ -129,19 +133,36 @@ class Pledge extends Component {
   }
   componentDidMount () {
     this.checkUserFields(this.props)
+
+    if (navigator) {
+      this.setState({ cookiesEnabled: navigator.cookieEnabled })
```

Comment:
The whole payment flow works without cookies from our side (e.g. a stripe purchase is possible without cookies, not sure about the others). I'd prefer to keep allowing this. Please remove from pledge form again. I'll make sure that this cookie error message becomes visible in all flows eventually (through a upcoming poller abstraction).

Reply:
Had same thought train going here.

Here's why I believe we should not allow a user walk through the payment process: While the payment flow works, the sing-in and confirmation procedure hereafter does not. As I understand, you can not complete the payment process fully nor use Republik hereafter if you haven't cookies enabled.

To prevent wrong notions, it feels more user-friendly (...) to not show checkout page.

Meanwhile, there would be that lovely option of listing what we would have to offer if you were inclined to enabled cookies: List pledges. This might still be option. I refrained from that due to my rudimentary frontend knowledge, speed of implementing it and "income/expense ratio".